.env.example

```diff
@@ -51,6 +51,36 @@ INVITATION_CODE=try-huginn
 # If you don't want to require new users to have an invitation code in order to sign up, set this to true.
 SKIP_INVITATION_CODE=false
 
+# If you require your users to confirm their email address after sign up set this to false.
+ALLOW_UNCONFIRMED_ACCESS=true
+
+# Duration in which the user needs to confirm the email address
+ALLOW_UNCONFIRMED_ACCESS_FOR=2.days
+
+# Duration for which the confirmation token is valid
+CONFIRM_WITHIN=3.days
+
+# Minimum password length
+MIN_PASSWORD_LENGTH=8
+
+# Duration for which the reset password token is valid
+RESET_PASSWORD_WITHIN=6.hours
+
+# When set to 'none' users will not be locked after MAX_FAILED_LOGIN_ATTEMPTS failed login attempts
```

Comment:
`Set to 'failed_attempts' to lock user accounts for the UNLOCK_AFTER period they fail MAX_FAILED_LOGIN_ATTEMPTS login attempts.  Set to 'none' to allow unlimited failed login attempts.`